### DISCUSSION TITLE
Export Scrypted Server Settings

### DISCUSSION THREAD

**Discussion by RodneyCapron** [koush/scrypted#265] [General]
Good Afternoon,

I'm in the process of moving away from running Scrypted (which is AMAZING!) on a Raspberry Pi to a Docker Container running on a QNAP NAS. Does anyone know if it is possible to export all the Scrypted server settings?

Thanks very much!

Rodney

**Comment by koush** [comment]
just move the volume at `~/.scrypted/volume` and it will transfer over fine.

**Comment by RodneyCapron** [comment]
Thank you for the quick reply @koush! I really appreciate it. 👍
